README.rst

```diff
@@ -24,6 +24,10 @@ consult https://tulip-control.sourceforge.io/doc/install.html
 The next section describes how to build documentation.
 A test suite is provided under ``tests/``.  Consult the section "Testing" below.
 
+Pip can install the latest development version too::
+
+  pip install https://github.com/tulip-control/tulip-control/archive/master.zip
```

Comment:
This is a useful command to install directly from GitHub. However, there is some risk of confusion by new users who copy-and-paste this command without noticing or understanding the significance of "latest development version".

My recommendation is to keep this command but add clarifying text; e.g.,

1. earlier in the README, note that `pip install tulip` will get the latest "release" or "current official release"; or
2. add a brief blob of text suggesting that things can break without warning if the latest development version is used, and that most users should first try instead to use `pip install tulip`.